src/PyMNE.jl

```diff
@@ -28,11 +28,13 @@ function __init__()
     # all of this is __init__() so that it plays nice with precompilation
     # see https://github.com/JuliaPy/PyCall.jl/#using-pycall-from-julia-modules
     copy!(mne, pyimport("mne"))
-    # delegate everything else to mne
-    for pn in propertynames(mne)
-        isdefined(@__MODULE__, pn) && continue
-        prop = getproperty(mne, pn)
-        @eval $pn = $prop
+    if ccall(:jl_generating_output, Cint, ()) == 0
```

Comment:
a comment could help give context for future readers, eg
```suggestion
    # fix https://github.com/beacon-biosignals/PyMNE.jl/issues/4
    if ccall(:jl_generating_output, Cint, ()) == 0
```